pkg/types/validation/installconfig.go

```diff
@@ -428,14 +428,33 @@ func validateControlPlane(platform *types.Platform, pool *types.MachinePool, fld
 	return allErrs
 }
 
+func validateComputeEdge(platform *types.Platform, p *types.MachinePool, fldPath *field.Path, pfld *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	switch platform.Name() {
```

Comment:
Is this meant to expand or can this really just be an If for now?

Reply:
> for now
for now, yes. Edge pool is supported only in AWS at this moment.